package.json

```diff
@@ -12,12 +12,13 @@
   ],
   "scripts": {
     "start": "tsdx watch",
-    "build": "tsdx build",
+    "build": "yarn apollo-codegen && tsdx build",
     "test": "tsdx test",
     "lint": "tsdx lint src test",
     "docs": "typedoc --out docs src/index.ts",
     "size": "size-limit",
-    "analyze": "size-limit --why"
+    "analyze": "size-limit --why",
+    "apollo-codegen": "apollo client:codegen --target typescript --endpoint=https://api.thegraph.com/subgraphs/name/tomprsn/yearn-vaults-v2-subgraph-mainnet --outputFlat --includes \"src/apollo/**\" --excludes \"src/apollo/generated\" --passthroughCustomScalars --customScalarsPrefix YearnSubgraph src/apollo/generated"
```

Comment:
Maybe it's worth writing a small Development section in the `README.md` to point out the need of running `apollo-codegen` to have correct typehints.